### PR TITLE
docs: update and add docs for date zoom

### PR DESCRIPTION
**Pull request by joaoviana**
Closes: #8039 

### Description:

Updates screenshots and adds docs on date zoom

I know other features like: `sharing dashboard with date zoom` aren't merged yet, but that can be added after!

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: docs/docs/guides/interactive-dashboards.mdx [hunk 1]
```diff
@@ -1,58 +1,114 @@
...
 
-The filters you add using cross-filtering won't be saved unless you're in `edit` mode (i.e. you've clicked `edit` in your dashboard) and you save the filters you applied. 
+The filters you add using cross-filtering won't be saved unless you're in `edit` mode (i.e. you've clicked `edit` in your dashboard) and you save the filters you applied.
 
 Otherwise, you can cross-filter away without any worry that you'll be changing the chart for everyone else! 😊
 
-### Drill into a metric in a chart or your results table using `drill by`
+## Date zoom
+
+When viewing a dashboard, you can also change the date granularity (e.g. to Week, Year) of the **charts with date dimensions in your dashboard** by clicking on the button `Date Zoom`. With this, you're able to quickly date-drill into your data without having to edit the chart.
```
Comment (TuringLovesDeathMetal) [inline docs/docs/guides/interactive-dashboards.mdx:63]:
```suggestion
When viewing a dashboard, you can also change the date granularity (e.g. to Week, Year) of the **charts with date dimensions in your dashboard** by clicking on the button `Date Zoom`. 

This is helpful if you want to zoom in to look more closely at something, or zoom out to identify a trend. You're able to quickly date-drill into your data without having to edit the chart.
```

File: docs/docs/guides/interactive-dashboards.mdx [hunk 2]
```diff
@@ -1,58 +1,114 @@
...
+
+When the date granularity of a chart is changed, you can hover over the indicator to see the date dimension that was affected, like so:
+
+<img
+    src={DateZoomIndicator}
+    width="400"
+    height="351"
+    style={{ display: 'block', margin: '0 auto 20px auto' }}
+/>
+
```
[the remark this answers is not in the document]
Comment (TuringLovesDeathMetal) [reply docs/docs/guides/interactive-dashboards.mdx:82]:
```suggestion
::: info

If you have more than one date dimension in your results, date drill will be applied to the first date dimension it scans - this is normally the left-most column in your results.

::: 
```

**Review by TuringLovesDeathMetal — CHANGES_REQUESTED** [review]
Looks amazing! And thanks so much for cleaning up loads of hte other bits of the doc ❤️ I've just added two little suggestions for a few more details around date zoom's functionality!

**Comment by joaoviana** [comment]
> Looks amazing! And thanks so much for cleaning up loads of hte other bits of the doc ❤️ I've just added two little suggestions for a few more details around date zoom's functionality!

and here it is! https://deploy-preview-8154--peaceful-bassi-cbf284.netlify.app/guides/interactive-dashboards

**Pull request merged**